GeosCore/dust_mod.F90

```diff
@@ -161,10 +165,11 @@ SUBROUTINE CHEMDUST( Input_Opt,  State_Chm, State_Diag, &
        ! NOTE: Ind_() returns -1 for species not found, so for the
        ! algorithm below to work, we need to make sure all id's are
        ! not less than zero (bmy, 7/5/16)
-       IF ( MAX( id_DST1, 0 )  + &
-            MAX( id_DST2, 0 )  + &
-            MAX( id_DST3, 0 )  + &
-            MAX( id_DST4, 0 ) == 0 ) THEN
+      ! extend 7 dust bins for deposition (D. Zhang, 28 Jun 2024)
+      IF ( MAX( id_DSTbin1, 0 ) +  MAX( id_DSTbin2, 0 ) +                    &
+           MAX( id_DSTbin3, 0 ) +  MAX( id_DSTbin4, 0 ) +                    &
+           MAX( id_DSTbin5, 0 ) +  MAX( id_DSTbin6, 0 ) +                    &
+           MAX( id_DSTbin7, 0 ) == 0                        ) THEN
```

Comment:
It looks like the indenting here is not quite right.

Reply:
@lizziel: Now fixed in commit a4fdcdcb.